app.js

```diff
@@ -184,6 +190,7 @@ var spOptions = {
   authnRequestBinding:          'HTTP-Redirect',
   validateInResponseTo:         true,
   privateCert:          argv.spPrivateKey,
+  additionalParams:{'RelayState':argv.relayState},
```

Comment:
Instead of specify in the start up parameter, it already support to specify the RelayState in the query parameter, for example, you can call /login?RelayState=testRelayState